### PR TITLE
Final changes to PPP and DG

### LINKED ISSUE #176: DG: Sequence diagram for data

### PR DESCRIPTION
**Pull request by clarencepohh**
Closes #176.

Sequence Diagram for Data has been added. 
PPP for myself has also been updated.

### CODE REVIEW

File: docs/diagrams/sequence/DataSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (NicholasTanYY) [inline docs/diagrams/sequence/DataSequenceDiagram.puml]: I think you might have forgotten to remove the spacing here!

File: docs/team/clarencepohh.md [hunk 2]
```diff
@@ -1,11 +1,65 @@
 # Clarence Poh Xiang Jun - Project Portfolio Page
+This page outlines what I have done for the module CS2113 - Software Engineering and OOP.
+
+Our project: CLI-nton, a powerful task management application designed to help you stay organized and productive. With a simple command-line interface, CLI-nton makes it easy to manage your tasks, whether they're todos, deadlines, or events.
+
+CLI-nton was an idea sprung by our group's collective minds, and it turned out to be a great practice in SWE as well as programming.
+Working with a new language - `Java` - helps to improve our tech stack (and hopefully our resumes!)
```
Comment (NicholasTanYY) [inline docs/team/clarencepohh.md:7]: Nice description of our project!

File: docs/team/clarencepohh.md [hunk 3]
```diff
@@ -1,11 +1,65 @@
...
+  - Alternatively, view the [GitHub Repo for our DG!](https://github.com/AY2324S2-CS2113-W13-2/tp/blob/master/docs/DeveloperGuide.md)
+
+- **Code Contribution**: [Reposense Link.](https://nus-cs2113-ay2324s2.github.io/tp-dashboard/?search=clarencepohh&breakdown=true&sort=groupTitle%20dsc&sortWithin=title&since=2024-02-23&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+
+- **Active Code Review**: Constantly monitoring build status of member's PRs, and try to assist in resolving them.
+  Here are some PRs reviewed by me (with non-trivial comments): [#43](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/43),
+  [#75](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/75), [77](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/77),
+  [#93](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/93), [#139](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/139),
+  [#147](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/147), [#148](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/148),
+  [#149](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/149), [#160](https://github.com/AY2324S2-CS2113-W13-2/tp/pull/160)
```
Comment (NicholasTanYY) [inline docs/team/clarencepohh.md:23]: Nice putting those relevant PR code reviews here!

File: docs/team/clarencepohh.md [hunk 4]
```diff
@@ -1,11 +1,65 @@
...
+        - Formatted the UG and created the Table of Contents according to content written by group mates.
+        - Maintained image consistency and proper organization of images in their respective folders.
+        - Alternatively, view the [GitHub Repo for our UG!](https://github.com/AY2324S2-CS2113-W13-2/tp/blob/master/docs/UserGuide.md)
+  
+  - Developer Guide:
+      - Formatted the DG and created the Table of Contents according to content written by group mates.
+      - Handled the overall flow of DG to ensure reading ease for developers.
+      - Wrote the Data component in the DG and created their respective UML diagrams [DG: Data Component](../DeveloperGuide.md/#data-component).
+      - Maintained the Appendix in the DG to reflect our application needs. See [DG: Appendix](../DeveloperGuide.md/#appendix-requirements). 
+        - Alternatively, view the [GitHub Repo for our DG!](https://github.com/AY2324S2-CS2113-W13-2/tp/blob/master/docs/DeveloperGuide.md)
```
Comment (NicholasTanYY) [inline docs/team/clarencepohh.md:51]: Great job filling in the documentation section!

File: docs/team/clarencepohh.md [hunk 5]
```diff
@@ -1,11 +1,65 @@
...
 
-### Summary of Contributions
-#### Created TaskManager and its related TaskManagerException classes
-#### Active PR reviewer
-#### Contributed JUnit tests to TaskManager and TaskManagerException classes
\ No newline at end of file
+- **Reflection**:
+  - Working on this project allowed me to understand what it would be like working in the future as a software developer.
+  I learnt plenty about the workflows used as well as tools that software engineers use in their day-to-day. The process 
+  was very enriching and I greatly enjoyed working with my teammates and solving problems together. I hope that the
```
Comment (Mahesh1772) [inline docs/team/clarencepohh.md:63]: Thank you for your hard work on this project too!

File: docs/team/clarencepohh.md [hunk 6]
```diff
@@ -1,11 +1,65 @@
...
-#### Created TaskManager and its related TaskManagerException classes
-#### Active PR reviewer
-#### Contributed JUnit tests to TaskManager and TaskManagerException classes
\ No newline at end of file
+- **Reflection**:
+  - Working on this project allowed me to understand what it would be like working in the future as a software developer.
+  I learnt plenty about the workflows used as well as tools that software engineers use in their day-to-day. The process 
+  was very enriching and I greatly enjoyed working with my teammates and solving problems together. I hope that the 
+  experience I have gained through this module will bring me further in my path, and that I will be able to be part of 
+  this style of workflows soon enough in the future! Thank you, CS2113! :)
```
Comment (NicholasTanYY) [inline docs/team/clarencepohh.md:65]: Wow, you have done a lot for this project! Great job Clarence!

**Review by NicholasTanYY — APPROVED** [review]
Overall LGTM!! This is ready for merging.

File: docs/team/clarencepohh.md [hunk 7]
```diff
@@ -1,11 +1,65 @@
...
+        - Alternatively, view the [GitHub Repo for our UG!](https://github.com/AY2324S2-CS2113-W13-2/tp/blob/master/docs/UserGuide.md)
+  
+  - Developer Guide:
+      - Formatted the DG and created the Table of Contents according to content written by group mates.
+      - Handled the overall flow of DG to ensure reading ease for developers.
+      - Wrote the Data component in the DG and created their respective UML diagrams [DG: Data Component](../DeveloperGuide.md/#data-component).
+      - Maintained the Appendix in the DG to reflect our application needs. See [DG: Appendix](../DeveloperGuide.md/#appendix-requirements). 
+        - Alternatively, view the [GitHub Repo for our DG!](https://github.com/AY2324S2-CS2113-W13-2/tp/blob/master/docs/DeveloperGuide.md)
+
+- **Community Contributions**:
```
Comment (Mahesh1772) [inline docs/team/clarencepohh.md:53]: Good job on providing extensive description of your part!

File: docs/images/sequence/DataSequenceDiagram.png
Comment (Mahesh1772) [inline docs/images/sequence/DataSequenceDiagram.png:1]: Thanks for the neat UML!

**Review by Mahesh1772 — COMMENTED** [review]
LGTM! Nice PPP

File: docs/diagrams/sequence/DataSequenceDiagram.puml [hunk 8]
```diff
@@ -0,0 +1,59 @@
+@startuml
```
Comment (kyhjonathan) [inline docs/diagrams/sequence/DataSequenceDiagram.puml:1]: Good job on creating the UML Sequence Diagram!

File: docs/diagrams/sequence/DataSequenceDiagram.puml [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        CH -> TM ++ : deleteManager(...)
+        note left: method parameters\n excluded for clarity
+            return
+        return
+    else rest omitted for clarity
+    end
+    return
+return await next input
+end
+@enduml
```
Comment (NicholasTanYY) [inline docs/diagrams/sequence/DataSequenceDiagram.puml:59]: Nice changes made to remove the additional lines!

**Review by NicholasTanYY — APPROVED** [review]
LGTM! Let's merge this now

**Pull request merged**